dev/bin/knative-installer.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 knative-setup.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+
+CONFIGURE_SCRIPT_DIR_PATH=`dirname "${BASH_SOURCE[0]}"`
+# The Knative Core APIs, like Broker or Trigger
```

Comment:
should we reflect that this is midstream?

Reply:
e.g. in comments?

Reply:
done :+1: